app/renderer/api/materialize.js

```diff
@@ -53,5 +53,11 @@ module.exports = {
     },
     closeModals() {
         $('#config-modal').modal('close');
+    },
+    getLengthNavBarTabs() {
+        return $('#navbar-tabs .tab-button').length
+    },
+    toggleNavBarMenu() {
+        $('#navBarMenu').click()
```

Comment:
I think materialize already provides a [method](http://materializecss.com/dropdown.html) to open/close dropdowns, It is cleaner using that instead of making a click